src/gui/transferlistmodel.cpp

```diff
@@ -450,7 +467,7 @@ QVariant TransferListModel::data(const QModelIndex &index, const int role) const
 
     switch (role) {
     case Qt::ForegroundRole:
-        return stateForeground(torrent->state());
+        return m_stateForegroundColors.value(torrent->state(), getColorByState(torrent->state()));
```

Comment:
This line confuses. It should be explicitly shown that m_stateForegroundColors provides colors from the theme. Otherwise it's unclear why do you have two color sources here.